experiment/measurer/measure_manager.py

```diff
@@ -438,9 +438,9 @@ def get_current_coverage(self) -> int:
                 self.cov_summary_file)
             coverage_data = coverage_info["data"][0]
             summary_data = coverage_data["totals"]
-            regions_coverage_data = summary_data["regions"]
-            regions_covered = regions_coverage_data["covered"]
-            return regions_covered
+            branches_coverage_data = summary_data['branches']
+            branches_covered = branches_coverage_data["covered"]
```

Comment:
nit: change this to single quotes please, it shouldn't have been double quotes to begin with.